webots/controllers/Supervisor/supervisor.py

```diff
@@ -73,51 +77,15 @@ def convert_webots_angle_to_ru_angle(webots_angle):
 
     webots_angle_2pi = convert_angle_to_2pi(webots_angle)
 
-    # TODO Handling the negative range.
+    # TO DO Handling the negative range.
 
     return webots_angle_2pi
 
-def has_position_changed(position, position_old):
-    """Returns whether the position changed.
-
-    Args:
-        position (list[float]): Position 1
-        position_old (list[float]): Position 2
-
-    Returns:
-        bool: If changed, it will return True otherwise False.
-    """
-    has_changed = False
-
-    for idx, value in enumerate(position):
-        if int(value) != int(position_old[idx]):
-            has_changed = True
-            break
-
-    return has_changed
 
-def has_orientation_changed(orientation, orientation_old):
-    """Returns whether the orientation changed.
-
-    Args:
-        orientation (list[float]): Orientation 1
-        orientation_old (list[float]): Orientation 2
-
-    Returns:
-        bool: If changed, it will return True otherwise False.
-    """
-    has_changed = False
-
-    for idx, value in enumerate(orientation):
-        if int(rad_to_deg(value)) != int(rad_to_deg(orientation_old[idx])):
-            has_changed = True
-            break
-
-    return has_changed
-
-class OdometryData(): # pylint: disable=too-few-public-methods
+class OdometryData():  # pylint: disable=too-few-public-methods
```

Comment:
Usage of dataclass would be appropriate for this container class. Might resolve pylint method finding.